config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
how about the rest of the properties that can be here? Is it only for demonstration or is it fully done?

Reply:
as far as i am concerned, everything important to be in config.json is already here.
But anyone can add anything later.